### PR TITLE
Algorithm that adds the main and minor diagonal of a matrix

### PR DESCRIPTION
**Pull request by YESENIA2012**
You can review it my love ?

### CODE REVIEW

**Comment by YESENIA2012** [comment]
You can check it thingy, please.

File: sumMainSecundaryDiagonal.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  let contador2 = 0;
+  console.log("Fila: ", contador);
+  while (contador2 < filaMatrix.length) {
+    let numeroInsertarColumnas = parseInt(lea("Elemento a insertar: "));
+    filaMatrix[contador2] = numeroInsertarColumnas;
+    contador2 = contador2 + 1;
+  }
+}
+console.log(matrix);
+*/
```
Comment (erick2014) [inline sumMainSecundaryDiagonal.js]: remove commented code from line 1 to line 21

File: sumMainSecundaryDiagonal.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+function sumDiagonals(matrix) {
+  var sumaDiagonalPrincipal = 0;
```
Comment (erick2014) [inline sumMainSecundaryDiagonal.js]: this should be declared with `let` to make it accessible only inside the function:
let sumPrimaryDiagonal = 0;
In english please, remember to change the variable inside the function where you are using it

File: sumMainSecundaryDiagonal.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  var sumaDiagonalSecundaria = 0;
```
Comment (erick2014) [inline sumMainSecundaryDiagonal.js]: this should be declared with `let` to make it accessible only inside the function:
let sumSecondaryDiagonal = 0;
in english please, remember to change the variable inside the function where you are using it

File: sumMainSecundaryDiagonal.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  return {
+    sumatoriaDiagonalPrincipal: sumaDiagonalPrincipal,
+    sumatoriaDiagonalSecundaria: sumaDiagonalSecundaria,
+  };
+}
+//var diagonales = sumDiagonals(matrix);
+
+//console.log(diagonales);
+//console.log("Principal", diagonales.variableDeSumatoria);
+//console.log("Secundaria", diagonales.variableDeSumatoriaDiagonalSecundaria);
```
Comment (erick2014) [inline sumMainSecundaryDiagonal.js]: remove commented code from lines 43 to 47

File: test/sumMainSecundaryDiagonal.test.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+var expect = require("chai").expect;
+
+const mainfunction = require("../sumMainSecundaryDiagonal");
+
+describe("Sum of main and secundary Diagonals", function () {
```
Comment (erick2014) [inline test/sumMainSecundaryDiagonal.test.js]: change the message to something like: "Add sumatory of primary and secondary diagonals"

File: test/sumMainSecundaryDiagonal.test.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  it("should show the sum between the main and secondary diagonal", function () {
```
Comment (erick2014) [inline test/sumMainSecundaryDiagonal.test.js]: "should show the sum of primary and secondary diagonals equal 10 "

File: test/sumMainSecundaryDiagonal.test.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    const result = mainfunction(matrix);
+    const result1 = result.sumatoriaDiagonalPrincipal;
+    const result2 = result.sumatoriaDiagonalSecundaria;
+
+    const expectedResult = 10;
+    const expectedResult2 = 10;
+    expect(result1).to.eql(expectedResult);
+    expect(result2).to.eql(expectedResult2);
+  });
+  it("it should show 0 as a result", function () {
```
Comment (erick2014) [inline test/sumMainSecundaryDiagonal.test.js]: add a break line on line 23 to separate the code

**Review by erick2014 — CHANGES_REQUESTED** [review]

**Comment by YESENIA2012** [comment]
Again I made the changes because it was wrong, the previous

**Pull request merged**